Zilon.Core/Zilon.Core/Tactics/Behaviour/HandMineDepositMethod.cs

```diff
@@ -0,0 +1,31 @@
+﻿using System;
+using System.Linq;
+
+using Zilon.Core.StaticObjectModules;
+
+namespace Zilon.Core.Tactics.Behaviour
+{
+    public sealed class HandMineDepositMethod : IMineDepositMethod
+    {
+        public HandMineDepositMethod()
+        {
+        }
+
+        public IMineDepositResult TryMine(IPropDepositModule deposit)
+        {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this redundant constructor.](https://app.codacy.com/manual/kreghek/Zilon_Roguelike/pullRequest?prid=5384689)